valhalla/mjolnir/osmway.h

```diff
@@ -2709,7 +2724,8 @@ struct OSMWay {
   uint16_t bike_backward_ : 1;
   uint16_t lit_ : 1;
   uint16_t destination_only_hgv_ : 1;
-  uint16_t spare2_ : 2;
+  uint16_t spare2_ : 1;
+  uint16_t multiple_levels_ : 1;
```

Comment:
better to have the spare last organizationally

```suggestion
  uint16_t multiple_levels_ : 1;
  uint16_t spare2_ : 1;
```